apps/app_manager.py

```diff
@@ -28,21 +32,20 @@ def __init__(self, app_directory, menu_class, printer_func, i, o):
         self.o = o
 
     def load_all_apps(self):
-        base_menu = self.menu_class([], self.i, self.o, "Main app menu", exitable=False) #Main menu for all applications.
+        base_menu = self.menu_class([], self.i, self.o, "Main app menu",
+                                    exitable=False)  # Main menu for all applications.
         base_menu.exit_entry = ["Exit", "exit"]
         base_menu.process_contents()
-        orderings = {}
         self.subdir_menus[self.app_directory] = base_menu
-        for path, subdirs, modules in app_walk(self.app_directory): 
-            #print("Loading path {} with modules {} and subdirs {}".format(path, modules, subdirs))
+        for path, subdirs, modules in app_walk(self.app_directory):
             for subdir in subdirs:
-                #First, we create subdir menus (not yet linking because they're not created in correct order) and put them in subdir_menus.
+                # First, we create subdir menus (not yet linking because they're not created in correct order) and put them in subdir_menus.
                 subdir_path = os.path.join(path, subdir)
                 self.subdir_menus[subdir_path] = self.menu_class([], self.i, self.o, subdir_path)
-            for module in modules:
-                #Then, we load modules and store them along with their paths
+            for _module in modules:
```

Comment:
Why the underscore? Just curious, doesn't seem to be a Python keyword or anything.

Reply:
Not a builtin keyword, but it is reserved as a type. 
```
import sys
type(sys)
> 'module'